styleguide/_themes/derek/scss/snowflakes/swatches.scss

```diff
@@ -0,0 +1,66 @@
+.swatch-color {
+  height: 100px;
+  //colors
+  &.mediumteal { @extend %bg-medium-teal; }
+  &.signup-green { background-color: $brand-success; }
+  &.magento-orange { background-color: $magento-orange; }
+  &.purple { background-color: $purple; }
+  &.mustard { background-color: $mustard; }
+
+  //grays
+  &.light-gray { background-color: $gray-lighter; }
+  &.medium-gray { background-color: $gray-base; }
+  &.medium-dark-gray { background-color: $gray-darker; }
+  &.darkest-gray { background-color: $gray-darkest; }
+
+  &:hover {
+
+    .swatch-copy-button {
+      @extend %center;
+      background-color: $gray-lightest;
+      display: block;
+      font-family: $subhead;
+      opacity: .85;
+      padding: 10px;
+      position: absolute;
+      text-transform: uppercase;
+      top: 60px;
+      width: 88%;
+    }
+  }
+}
+
+.swatch-color-lighter {
+  height: 40px;
+  position: absolute;
+  right: 15px;
+  top: 0;
+  width: 40px;
+  &.signup-green { background-color: lighten($brand-success, 5%); }
+  &.mustard { background-color: lighten($mustard, 5%); }
```

Comment:
We can change that. Thats just what brand calls that color. I'm fine to branch off if you think that's a better naming options though at that point I may argue going for its pantone name.